homeassistant/components/emulated_roku/binding.py

```diff
@@ -155,7 +155,7 @@ async def emulated_roku_start(event):
                 )
 
         # start immediately if already running
-        if self.hass.state == CoreState.running:
+        if self.hass.state is CoreState.running:
```

Comment:
Side note: Most of these places should use one of our start helpers, `async_at_start` or `async_at_started`.